internal/driver/commands.go

```diff
@@ -59,14 +60,6 @@ func (c *command) help(name string) string {
 	return message + "\n"
 }
 
-// AddCommand adds an additional command to the set of commands
-// accepted by pprof. This enables extensions to add new commands for
-// specialized visualization formats. If the command specified already
-// exists, it is overwritten.
-func AddCommand(cmd string, format int, post PostProcessor, desc, usage string) {
```

Comment:
This code is also used in some environments where the internal package rules are not enforced.